workflow/rules/common.smk

```diff
@@ -120,9 +120,9 @@ def get_fq(wildcards):
                 )
             )
         if not is_paired_end(wildcards.sample):
-            return {"fq1": f"{u.fq1}"}
+            return {"fq1": u.fq1.to_list()}
```

Comment:
`u.fq1` should always only have one entry, right? So I would also go for the `squeeze()` solution, here. And additionally make it the same column selection syntax as above:
```suggestion
            return {"fq1": f"{u['fq1'].squeeze()"}
````